app/Http/Controllers/App/Bookkeeping/ReceiptController.php

```
@@ -146,6 +146,25 @@ public function printReport(Request $request): BinaryFileResponse
         return response()->inlineFile($pdf, $filename);
     }
 
+    public function printCostcenterReport(Request $request): BinaryFileResponse
+    {
+
+        $query = Receipt::query();
+        $this->applyReceiptQueryFilters($query, $request, $search);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$search'. (undefined)

(UnusedLocalVariable)